app/manage-breast-screening/2025/11/ending-breast-screening-appointments/index.md

```diff
@@ -0,0 +1,133 @@
+
+![Screenshot of a form providing options for rescheduling the appointment](reschedule-info.png)
+
+This reflects the limitations of the current booking system. We're using this as a starting point, while working with colleagues rebuilding the screening invitation system to ensure any options provided here are in line with future functionality. 
+
+## 2. Stopping the appointment after it starts
+
+Once the mammographer conducting the screening has actively begun the appointment workflow, two statuses are possible; `Attended not screened` (ANS) and `Partial mammogram`.
+
+The partial mammogram scenario covers any instance where at least one X-ray has been taken, but the appointment could not be concluded. This is a more complicated journey that we are tackling separately.
+
+The reasons for a participant to be marked as ANS include: 
+
+* they decide to opt out
+* they don't understand instructions given to them
+* they fail the ID check in the mammogram room
+* they've had a total bilateral mastectomy
+* there are technical issues with equipment before any images are taken
+
+At each stage of the workflow, an 'Appointment cannot proceed' link is available. When users select this they are asked for a reason. This list is very similar to the 'Cancel appointment' options, but has been adapted to include things that can only occur once the appointment has actually begun.
+
+![Screenshot of a form with reasons for stopping the appointment after it has begun and options for rescheduling](appointment-cannot-proceed.png)
+
+As well as actively stopping an appointment, users may reach an end point based on other information they enter into the service.
+
+### Recent mammograms
+
+For example, if the participant tells them about a mammogram that took place within the last 6 months, they would be advised not to go ahead.
+
+![Screenshot of a conditional page that appears when a recent mammogram has been recorded](recent-mammogram.png)
+
+### Withdrawn consent
+
+If the participant does not consent to mammograms of breast implants, users would be given options for how to end the appointment.
+
+![Screenshot of a conditional page that appears when consent to continue has not been given](no-consent.png)
+
+### Bilateral mastectomy
+
+If the participant tells the mammographer they have no breast tissue remaining due to a full double mastectomy, we will offer the user the option to end the appointment if no images need to be taken.
+
+## 3. Handling outstanding appointments after the clinic ends
+
+The only status chance that can be applied here is `Did not attend` (DNA). This is where the participant does not turn up for their appointment and does not make contact to actively cancel it. 
+
+We've not yet designed the 'Close clinic' journey, but if any open appointments exist at the end of a clinic we need to tell users about any DNA statuses they are about to create.
```

Comment:
```suggestion
We've not yet designed a 'Close clinic' journey, but if any open appointments exist at the end of a clinic we need to tell users about any DNA statuses they are about to create.
```

I'm not sure it would be obvious what a 'close clinic' journey means. Could we elaborate about what it is and what the current process is?